docs/references/what_is_a_cve.md

```diff
@@ -0,0 +1,11 @@
+# What is a CVE
+
+A CVE (Common Vulnerabilities and Exposures) is a way to catalog and track public security
```

Comment:
```suggestion
Common Vulnerabilities and Exposures (CVEs) are a way to catalogue and track public security
```

Reply:
It's generally better to put acronyms and their expansions this way around so that you're introducing the concept first (and then the name you're going to call it by)

Reply:
Also, should we include this link? https://ubuntu.com/security/cves

Reply:
Good point, I will add a paragraph for that